### DISCUSSION TITLE
faster-MMS-ASR models

### DISCUSSION THREAD

**Discussion by allandclive** [SYSTRAN/faster-whisper#393] [Ideas]
MMS-ASR has a ton of low resource languages that Whisper doesn't support. It would be helpful to develop one for these languages

**Comment by guillaumekln** [comment]
MMS has a non commercial license, so it is less interesting than Whisper in that regard.

**Comment by silvacarl2** [comment]
based on preliminary testing, its not nearly as good as faster-whisper.
